src/gui/guiFormSpecMenu.cpp

```diff
@@ -3296,6 +3350,11 @@ void GUIFormSpecMenu::drawMenu()
 		e->setVisible(false);
 	}
 
+	/*
+		Update animated images
+	*/
+	m_texture_pool.step();
+
```

Comment:
This should probably be done a bit earlier, before anything is drawn, eg. in line 3316.